OutOfSchool/OutOfSchool.Common/Responses/ApiError.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace OutOfSchool.Common.Responces;
```

Comment:
(Mistype) Namespace `Responces` should contain no "c" in it's name. Please fix it